CIP-????/README.md

```diff
@@ -0,0 +1,132 @@
+2. Issuing governance tokens - each user gets a vote on where the pooled assets will be delegated.
+
+This general pattern is utilized by all kinds of L1 Dapps: DAOs, DEXs, p2p lending, etc. Unfortunately, the above mitigations do not fully solve the issue; the blockchain is still significantly more centralized than if full delegation control was used.
+
+## Specification
+
+### Beacon Tokens
+
+#### What are Beacon Tokens?
+All native tokens on Cardano broadcast their current address and any transactions they were ever a part of at all times. Take a look at the following off-chain APIs:
+
+| Task | Koios API | Blockfrost API |
+|--|--|--|
+| Addresses with asset | [API](https://API.koios.rest/#get-/asset_address_list) | [API](https://docs.blockfrost.io/#tag/Cardano-Assets/paths/~1assets~1%7Basset%7D~1addresses/get) |
+| Txs including asset | [API](https://API.koios.rest/#get-/asset_txs) | [API](https://docs.blockfrost.io/#tag/Cardano-Assets/paths/~1assets~1%7Basset%7D~1transactions/get) |
+
+Knowing *exactly* which addresses/transactions have the token, you can then use the same off-chain API services to take a closer look at those addresses/transactions:
+
+| Task | Koios API | Blockfrost API |
+|--|--|--|
+| UTxO info of address | [API](https://API.koios.rest/#post-/address_info) | [API](https://docs.blockfrost.io/#tag/Cardano-Addresses/paths/~1addresses~1%7Baddress%7D~1utxos/get)|
+| Tx metadata | [API](https://API.koios.rest/#post-/tx_metadata) | [API](https://docs.blockfrost.io/#tag/Cardano-Transactions/paths/~1txs~1%7Bhash%7D~1metadata/get)|
+
+Here is a short list of what you can get:
+
+1. Any reference scripts stored at the address and their associated UTxO TxIx.
+2. Any datums stored at the address.
+3. The stake address associated to that payment address.
+4. The metadata history of each transaction that had the token.
+5. The datum history of each UTxO that held the token.
+
+These queries are possible for *all* Cardano native tokens. However, while all native tokens *can* broadcast this information, the broadcasting itself is usually not the intended purpose of the native token (eg, governance tokens). **A *Beacon Token* is a native token whose ONLY purpose is to broadcast information.** If you want to broadcast certain transactions, just make sure the beacon token is part of those transactions. If you want to broadcast certain addresses, just make sure the beacon token is stored at those addresses. If you want to broadcast certain UTxOs, just make sure the beacon token is stored in those UTxOs.
+
+#### Using Beacon Tokens
+Every native token has two configurable fields: the policy ID and the token name. The policy ID will be application specific while the token name can be data specific. To use a beacon token, simply create a native token that is unique to your application and if necessary, control how UTxOs with beacons can be spent.
+
+Here are a few basic reference implementations:
```

Comment:
These are good reference implementations but they're mentioned like anecdotes instead of a technically readable process of how the Beacon Tokens are used... as CIPs normally have in their Specification section.

@SebastienGllmt mentioned this in the meeting & maybe he could explain more about how this could be done to achieve a more rigorously written proposal.

Reply:
Beacon Tokens are highly generalizable, so we must find a balance in specifying how they should be used; without being overly vague or too specific. To that end, we figure it'd be worthwhile to wait until a few more reference implementations of *distributed dApps* that use beacon tokens are ready (currently being worked on), so that we can reference them in the CIP and extract from them a general pattern of Beacon Token usage.
@fallen-icarus